### PR TITLE
Use `steps.draft.outputs.body`

### PR DESCRIPTION
**Pull request by jglick**
https://github.com/jenkins-infra/github-reusable-workflows/pull/1#issuecomment-1135852603 alternative to #2 using https://github.com/jenkins-infra/interesting-category-action/pull/5 as per https://github.com/jenkins-infra/github-reusable-workflows/pull/2#issuecomment-1145526728.

### CODE REVIEW

File: .github/workflows/maven-cd.yml [hunk 1]
```diff
@@ -35,11 +36,12 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Check interesting categories
-        uses: jenkins-infra/interesting-category-action@v1.1.0
+        uses: jetersen/interesting-category-action@patch-1 # TODO https://github.com/jenkins-infra/interesting-category-action/pull/5
```
Comment (jglick) [inline .github/workflows/maven-cd.yml]: pending release

**Comment by jglick** [comment]
@jetersen squash-merged #1, making this more complicated…

**Comment by jetersen** [comment]
`git pull --rebase origin main` makes it super easy 🤔

**Comment by jglick** [comment]
I used some `git-rebase` variant but it was complicated by the base branch going away.

**Comment by jglick** [comment]
https://github.com/jenkinsci/jenkins-infra-test-plugin/pull/23 was correctly ignored; now checking that https://github.com/jenkinsci/jenkins-infra-test-plugin/pull/24 correctly triggers a release.

**Comment by jetersen** [comment]
https://github.com/jenkinsci/jenkins-infra-test-plugin/runs/6869048725?check_suite_focus=true seems to work @jglick

**Comment by jetersen** [comment]
Fixed the build failure by using a version bom that had 2.249.x bom:

https://github.com/jenkinsci/jenkins-infra-test-plugin/commit/49b5815e8158d5e5db9ee94039f23d12c1a7e63c my bad for not making a PR 😐 Though this commit still triggered a release so win/win? 😅 
https://github.com/jenkinsci/jenkins-infra-test-plugin/releases/tag/69.v49b_5815e8158

**Comment by jglick** [comment]
https://github.com/jenkinsci/bom/releases?page=3 huh, I seem to have picked a version from my local repo that was in `incrementals` but not `releases`. At least this demonstrated that https://github.com/jenkins-infra/jenkins-maven-cd-action/issues/3 is fixed!

**Comment by jglick** [comment]
> this commit still triggered a release

Yes, any commit could trigger a release if the currently unreleased changes include at least one PR with an interesting label.

**Comment by jetersen** [comment]
@jglick you should be able to use the github's release page on https://github.com/jenkins-infra/interesting-category-action/releases to draft a new release and pick a tag like `v1.2.0` ? 👏
EDIT: whelp just as fast I as I could type it, you created a release 😆

**Comment by jglick** [comment]
> use the github's release page

Actually easier:

```bash
gh release create v1.2.0 --generate-notes
```

**Review by jetersen — APPROVED** [review]

**Comment by jetersen** [comment]
@jglick cool! Did not know that CLI options existed!

**Comment by jglick** [comment]
So what is the next step here? We need an initial release tag. Then I can update the test plugin to use that as a final sanity check. Then we should amend the workflow template in `.github` to suggest it, and start rolling it out to various plugins?

**Comment by jetersen** [comment]
Yea @jglick :)

I suggest we create a v1.0.0 and a v1

and than move v1 along v1.x.x and revert v1 if we have any bad v1.x.x releases.

**Pull request merged**